### ISSUE TITLE
usage for hybrid/metagga bandstructure unfolding

### ISSUE THREAD

**Issue by alpinnovianus** [QijingZheng/VaspBandUnfolding#20]
How to use for hybrid/metagga band structure unfolding? 
(KPOINTS: weighted IBZKPT points + zero-weighted band path k-points)

is it enough to just set the weight of the generated k-points made by this script to zero?

<img width="778" alt="image" src="https://user-images.githubusercontent.com/74560218/210228922-404c3ddc-21eb-459c-b920-2fdd7bee29a2.png">

how should we do about the post-processing step with such KPOINTS?

**Comment by QijingZheng** [comment]
Yes, just set the weight of the extra k-points generated by this script to zero. No special actions in the post-processing is needed.
